pkg/controller/prpqr_reconciler/prpqr_reconciler_test.go

```diff
@@ -204,10 +205,26 @@ func prunePRPQRForTests(items []v1.PullRequestPayloadQualificationRun) {
 	}
 }
 
-func pruneProwjobsForTests(items []prowv1.ProwJob) {
+func pruneProwjobsForTests(t *testing.T, items []prowv1.ProwJob) {
 	for i, pj := range items {
 		if strings.HasPrefix(pj.Spec.Job, "aggregator") {
-			items[i].Spec.PodSpec.Containers[0].Args[5] = "--job-start-time=some-time"
+			unResolvedConfig := items[i].Spec.PodSpec.Containers[0].Env[0].Value
+
+			c := &api.ReleaseBuildConfiguration{}
+			if err := yaml.Unmarshal([]byte(unResolvedConfig), c); err != nil {
+				t.Fatal(err)
+			}
+
+			if _, ok := c.Tests[0].MultiStageTestConfiguration.Environment["JOB_START_TIME"]; ok {
+				c.Tests[0].MultiStageTestConfiguration.Environment["JOB_START_TIME"] = "1970-01-01T01:00:00+01:00"
```

Comment:
Why is this necessary? Surely you could either make the assertion logic later smarter, or use an injectable fixture for `time` so the test can set it?